src/util/ansi_color.h

```diff
@@ -0,0 +1,13 @@
+/*
+    Use ANSI 8/16 codes to output coloured text 
+    to the terminal in Linux/Mac/Windows.
+    NOTE: The colors can vary depending of the terminal configuration.
+    See https://misc.flogisoft.com/bash/tip_colors_and_formatting
+*/
+
```

Comment:
You can use fmt library to handle the colors ``fmt::print(fg(fmt::color::crimson) | fmt::emphasis::bold, "Hello, {}!\n", "world");``